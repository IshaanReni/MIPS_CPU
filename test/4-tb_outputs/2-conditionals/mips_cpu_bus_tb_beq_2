RAM : INIT : Loading RAM contents from ./test/2-bin/2-conditionals/beq_2.hex.txt
RAM : INIT : RAM contents at 00004e20 are: 24
RAM : INIT : RAM contents at 00004e21 are: 63
RAM : INIT : RAM contents at 00004e22 are: 00
RAM : INIT : RAM contents at 00004e23 are: 14
RAM : INIT : RAM contents at 00004e24 are: 24
RAM : INIT : RAM contents at 00004e25 are: 84
RAM : INIT : RAM contents at 00004e26 are: 00
RAM : INIT : RAM contents at 00004e27 are: 0a
RAM : INIT : RAM contents at 00004e28 are: 10
RAM : INIT : RAM contents at 00004e29 are: 22
RAM : INIT : RAM contents at 00004e2a are: 00
RAM : INIT : RAM contents at 00004e2b are: 03
RAM : INIT : RAM contents at 00004e2c are: 00
RAM : INIT : RAM contents at 00004e2d are: 40
RAM : INIT : RAM contents at 00004e2e are: 18
RAM : INIT : RAM contents at 00004e2f are: 21
RAM : INIT : RAM contents at 00004e30 are: 00
RAM : INIT : RAM contents at 00004e31 are: 42
RAM : INIT : RAM contents at 00004e32 are: 20
RAM : INIT : RAM contents at 00004e33 are: 21
RAM : INIT : RAM contents at 00004e34 are: 00
RAM : INIT : RAM contents at 00004e35 are: 42
RAM : INIT : RAM contents at 00004e36 are: 10
RAM : INIT : RAM contents at 00004e37 are: 21
RAM : INIT : RAM contents at 00004e38 are: 00
RAM : INIT : RAM contents at 00004e39 are: 00
RAM : INIT : RAM contents at 00004e3a are: 00
RAM : INIT : RAM contents at 00004e3b are: 08
RAM : INIT : RAM contents at 00004e3c are: 00
RAM : INIT : RAM contents at 00004e3d are: 00
RAM : INIT : RAM contents at 00004e3e are: 00
RAM : INIT : RAM contents at 00004e3f are: 00
RAM : INIT : RAM contents at 00004e40 are: 00
RAM : INIT : RAM contents at 00004e41 are: 00
RAM : INIT : RAM contents at 00004e42 are: 00
RAM : INIT : RAM contents at 00004e43 are: 00
RAM : INIT : RAM contents at 00004e44 are: 00
RAM : INIT : RAM contents at 00004e45 are: 00
RAM : INIT : RAM contents at 00004e46 are: 00
RAM : INIT : RAM contents at 00004e47 are: 00
RAM : INIT : RAM contents at 00004e48 are: 00
RAM : INIT : RAM contents at 00004e49 are: 00
RAM : INIT : RAM contents at 00004e4a are: 00
RAM : INIT : RAM contents at 00004e4b are: 00
RAM : INIT : RAM contents at 00004e4c are: 00
RAM : INIT : RAM contents at 00004e4d are: 00
RAM : INIT : RAM contents at 00004e4e are: 00
RAM : INIT : RAM contents at 00004e4f are: 00
RAM : INIT : RAM contents at 00004e50 are: 00
RAM : INIT : RAM contents at 00004e51 are: 00
RAM : INIT : RAM contents at 00004e52 are: 00
RAM : INIT : RAM contents at 00004e53 are: 00
RAM : INIT : RAM contents at 00004e54 are: 00
RAM : INIT : RAM contents at 00004e55 are: 00
RAM : INIT : RAM contents at 00004e56 are: 00
RAM : INIT : RAM contents at 00004e57 are: 00
RAM : INIT : RAM contents at 00004e58 are: 00
RAM : INIT : RAM contents at 00004e59 are: 00
RAM : INIT : RAM contents at 00004e5a are: 00
RAM : INIT : RAM contents at 00004e5b are: 00
RAM : INIT : RAM contents at 00004e5c are: 00
RAM : INIT : RAM contents at 00004e5d are: 00
RAM : INIT : RAM contents at 00004e5e are: 00
RAM : INIT : RAM contents at 00004e5f are: 00
RAM : INIT : RAM contents at 00004e60 are: 00
RAM : INIT : RAM contents at 00004e61 are: 00
RAM : INIT : RAM contents at 00004e62 are: 00
RAM : INIT : RAM contents at 00004e63 are: 00
RAM : INIT : RAM contents at 00004e64 are: 00
RAM : INIT : RAM contents at 00004e65 are: 00
RAM : INIT : RAM contents at 00004e66 are: 00
RAM : INIT : RAM contents at 00004e67 are: 00
RAM : INIT : RAM contents at 00004e68 are: 00
RAM : INIT : RAM contents at 00004e69 are: 00
RAM : INIT : RAM contents at 00004e6a are: 00
RAM : INIT : RAM contents at 00004e6b are: 00
RAM : INIT : RAM contents at 00004e6c are: 00
RAM : INIT : RAM contents at 00004e6d are: 00
RAM : INIT : RAM contents at 00004e6e are: 00
RAM : INIT : RAM contents at 00004e6f are: 00
RAM : INIT : RAM contents at 00004e70 are: 00
RAM : INIT : RAM contents at 00004e71 are: 00
RAM : INIT : RAM contents at 00004e72 are: 00
RAM : INIT : RAM contents at 00004e73 are: 00
RAM : INIT : RAM contents at 00004e74 are: 00
RAM : INIT : RAM contents at 00004e75 are: 00
RAM : INIT : RAM contents at 00004e76 are: 00
RAM : INIT : RAM contents at 00004e77 are: 00
RAM : INIT : RAM contents at 00004e78 are: 00
RAM : INIT : RAM contents at 00004e79 are: 00
RAM : INIT : RAM contents at 00004e7a are: 00
RAM : INIT : RAM contents at 00004e7b are: 00
RAM : INIT : RAM contents at 00004e7c are: 00
RAM : INIT : RAM contents at 00004e7d are: 00
RAM : INIT : RAM contents at 00004e7e are: 00
RAM : INIT : RAM contents at 00004e7f are: 00
RAM : INIT : RAM contents at 00004e80 are: 00
RAM : INIT : RAM contents at 00004e81 are: 00
RAM : INIT : RAM contents at 00004e82 are: 00
RAM : INIT : RAM contents at 00004e83 are: 00
REG : Register value changed to: xxxxxxxx, Location : xx
CPU : INFO   : Read instruction=xxxxxxxx
CPU : INFO   : Read instruction=xxxxxxxx
CPU : INFO   : Resetting.
CPU : INFO   : Fetching instruction=xxxxxxxx
CPU : INFO   : Read instruction=xxxxxxxx
CPU : INFO   : Resetting.
CPU : INFO   : Fetching instruction=xxxxxxxx
CPU : INFO   : Read instruction=xxxxxxxx
CPU : INFO   : Resetting.
CPU : INFO   : Fetching instruction=xxxxxxxx
TB : Reset is now low.
CTRL : instruction word: xxxxxxxx
CPU : INFO   : Resetting.
CPU : INFO   : Fetching instruction=xxxxxxxx
CPU : INFO   : Read instruction=bfc00000
CPU : INFO   : Read instruction=bfc00000
CPU : INFO   : Read instruction=bfc00000
CPU : INFO   : Read instruction=bfc00000
RAM: reading data at location: bfc00000  data: 00000000
CTRL : instruction word: xxxxxxxx
CPU : INFO   : Read instruction=bfc00000
CPU : INFO   : Read instruction=bfc00000
CPU : INFO   : Read instruction=bfc00000
RAM: reading data at location: bfc00000  data: 14006324
CTRL : instruction word: 00000000
REG : Register value changed to: xxxxxxxx, Location : 00
CTRL : got inside the i-type.
CPU : INFO   : Read instruction=bfc00008
CPU : INFO   : Read instruction=bfc00008
CPU : INFO   : Read instruction=bfc00008
RAM: reading data at location: bfc00008  data: 14006324
CTRL : instruction word: 24630014
REG : Register value changed to: xxxxxxxx, Location : 00
CTRL : got inside the i-type.
CPU : INFO   : Read instruction=bfc00010
CPU : INFO   : Read instruction=bfc00010
CPU : INFO   : Read instruction=bfc00010
RAM: reading data at location: bfc00010  data: 03002210
CTRL : instruction word: 24630014
REG : Register value changed to: xxxxxxxx, Location : 00
CTRL : got inside the i-type.
CTRL : got inside the branch.
CPU : INFO   : Read instruction=bfc00018
CPU : INFO   : Read instruction=bfc00018
CPU : INFO   : Read instruction=bfc00018
RAM: reading data at location: bfc00018  data: 21204200
CTRL : instruction word: 10220003
CPU : INFO   : Read instruction=bfc00020
CPU : INFO   : Read instruction=bfc00020
CPU : INFO   : Read instruction=bfc00020
RAM: reading data at location: bfc00020  data: 08000000
CTRL : instruction word: 00422021
REG : Register value changed to: ffffffec, Location : 04
CTRL : got inside the j-type.
CPU : INFO   : Read instruction=bfc00028
CPU : INFO   : Read instruction=bfc00028
CPU : INFO   : Read instruction=bfc00028
RAM: reading data at location: bfc00028  data: 00000000
CTRL : instruction word: 00000008
REG : Register value changed to: ffffffec, Location : 00
CPU : INFO   : Read instruction=bfc00030
CPU : INFO   : Read instruction=bfc00030
CPU : INFO   : Read instruction=bfc00030
RAM: reading data at location: bfc00030  data: 00000000
CTRL : instruction word: 00000000
REG : Register value changed to: ffffffec, Location : 00
CPU : INFO   : Read instruction=00000000
CPU : INFO   : Read instruction=00000000
CPU : INFO   : Read instruction=00000000
TB : finished; running=0
TB : INFO : register_v0=00000000
test/0-MIPS_tb_RAM/mips_cpu_bus_tb.v:95: $finish called at 55000 (10ps)
          0
