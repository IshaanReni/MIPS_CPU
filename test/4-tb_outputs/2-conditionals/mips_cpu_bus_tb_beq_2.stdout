RAM : INIT : Loading RAM contents from ./test/2-bin/2-conditionals/beq_2.hex.txt
RAM : INIT : RAM contents at 00004e20 are: 24
RAM : INIT : RAM contents at 00004e21 are: 63
RAM : INIT : RAM contents at 00004e22 are: 00
RAM : INIT : RAM contents at 00004e23 are: 14
RAM : INIT : RAM contents at 00004e24 are: 24
RAM : INIT : RAM contents at 00004e25 are: 84
RAM : INIT : RAM contents at 00004e26 are: 00
RAM : INIT : RAM contents at 00004e27 are: 0a
RAM : INIT : RAM contents at 00004e28 are: 10
RAM : INIT : RAM contents at 00004e29 are: 22
RAM : INIT : RAM contents at 00004e2a are: 00
RAM : INIT : RAM contents at 00004e2b are: 03
RAM : INIT : RAM contents at 00004e2c are: 00
RAM : INIT : RAM contents at 00004e2d are: 03
RAM : INIT : RAM contents at 00004e2e are: 10
RAM : INIT : RAM contents at 00004e2f are: 21
RAM : INIT : RAM contents at 00004e30 are: 00
RAM : INIT : RAM contents at 00004e31 are: 44
RAM : INIT : RAM contents at 00004e32 are: 10
RAM : INIT : RAM contents at 00004e33 are: 21
RAM : INIT : RAM contents at 00004e34 are: 00
RAM : INIT : RAM contents at 00004e35 are: 42
RAM : INIT : RAM contents at 00004e36 are: 10
RAM : INIT : RAM contents at 00004e37 are: 21
RAM : INIT : RAM contents at 00004e38 are: 00
RAM : INIT : RAM contents at 00004e39 are: 00
RAM : INIT : RAM contents at 00004e3a are: 00
RAM : INIT : RAM contents at 00004e3b are: 08
RAM : INIT : RAM contents at 00004e3c are: 00
RAM : INIT : RAM contents at 00004e3d are: 00
PC: not reset or halt
FSM: FETCH
CTRL: Here is the result =          0
CTRL: Read instruction at RAM address: xxxxxxxx
FSM: EXEC1
TB : Reset is now low.
CTRL: RAM INFO DATA READ: 00000000
CTRL : instruction word: 00000000, opcode: 000000, funccode: 000000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CPU : INFO   : Resetting.
CTRL: RAM INFO DATA READ: 00000000
CTRL : instruction word: 00000000, opcode: 000000, funccode: 000000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: SLL SHAMT 00, Instruction:00000000, FUNCCODE:000000
HHHHHHHHH ALU CODE 00
CTRL: RAM INFO DATA READ: 00000000
CTRL : instruction word: 00000000, opcode: 000000, funccode: 000000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: SLL SHAMT 00, Instruction:00000000, FUNCCODE:000000
HHHHHHHHH ALU CODE 00
ALU: ALU input A:         x
ALU: ALU input B:         x
ALU: ALU opcode: 0
ALU: ALU r:         x
CTRL: RAM INFO DATA READ: 00000000
CTRL : instruction word: 00000000, opcode: 000000, funccode: 000000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: SLL SHAMT 00, Instruction:00000000, FUNCCODE:000000
HHHHHHHHH ALU CODE 00
PC: not reset or halt
FSM: FETCH
CTRL: Here is the result =          x
CTRL: Read instruction at RAM address: bfc00000
FSM: EXEC1
PC: not reset or halt
CTRL: RAM INFO DATA READ: 14006324
CTRL : instruction word: 24630014, opcode: 001001, funccode: 010100
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: RAM INFO DATA READ: 14006324
CTRL : instruction word: 24630014, opcode: 001001, funccode: 010100
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: INFO: ADDIU called
CTRL: INPUT INFO: Rs = 03 Rt = 03 Immediate = 0014 ALUCODE = 09 Rd = 03
PC: not reset or halt
FSM: EXEC2
CTRL: Instruction again: 00100100011000110000000000010100, FUNCCODE: 010100
CTRL: Here is the RsDATA in andi =          0
CTRL: Here is the result =          x
CTRL: Instruction again: 00100100011000110000000000010100, FUNCCODE: 010100
CTRL: Here is the RsDATA in andi =          0
CTRL: Here is the result =         20
FSM: FETCH
PC: not reset or halt
PC: current value before increment: bfc00000
CTRL: Here is the result =         20
CTRL: Read instruction at RAM address: bfc00000
CTRL: Here is the result =         20
CTRL: Read instruction at RAM address: bfc00004
PC: not reset or halt
FSM: EXEC1
CTRL: RAM INFO DATA READ: 0a008424
CTRL : instruction word: 2484000a, opcode: 001001, funccode: 001010
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: INFO: ADDIU called
CTRL: INPUT INFO: Rs = 04 Rt = 04 Immediate = 000a ALUCODE = 09 Rd = 04
CTRL: RAM INFO DATA READ: 0a008424
CTRL : instruction word: 2484000a, opcode: 001001, funccode: 001010
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: INFO: ADDIU called
CTRL: INPUT INFO: Rs = 04 Rt = 04 Immediate = 000a ALUCODE = 09 Rd = 04
CTRL: RAM INFO DATA READ: 0a008424
CTRL : instruction word: 2484000a, opcode: 001001, funccode: 001010
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: INFO: ADDIU called
CTRL: INPUT INFO: Rs = 04 Rt = 04 Immediate = 000a ALUCODE = 09 Rd = 04
FSM: EXEC2
PC: not reset or halt
CTRL: Instruction again: 00100100100001000000000000001010, FUNCCODE: 001010
CTRL: Here is the RsDATA in andi =         20
CTRL: Here is the result =         20
CTRL: Instruction again: 00100100100001000000000000001010, FUNCCODE: 001010
CTRL: Here is the RsDATA in andi =         20
CTRL: Here is the result =         30
CTRL: Instruction again: 00100100100001000000000000001010, FUNCCODE: 001010
CTRL: Here is the RsDATA in andi =          0
CTRL: Here is the result =         30
CTRL: Instruction again: 00100100100001000000000000001010, FUNCCODE: 001010
CTRL: Here is the RsDATA in andi =          0
CTRL: Here is the result =         10
PC: not reset or halt
PC: current value before increment: bfc00004
FSM: FETCH
CTRL: Here is the result =         10
CTRL: Read instruction at RAM address: bfc00004
CTRL: Here is the result =         10
CTRL: Read instruction at RAM address: bfc00008
FSM: EXEC1
PC: not reset or halt
CTRL: RAM INFO DATA READ: 03002210
CTRL : instruction word: 10220003, opcode: 000100, funccode: 000011
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: INFO: ADDIU called
CTRL: INPUT INFO: Rs = 01 Rt = 02 Immediate = 0003 ALUCODE = 04 Rd = 01
CTRL: RAM INFO DATA READ: 03002210
CTRL : instruction word: 10220003, opcode: 000100, funccode: 000011
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: INFO: ADDIU called
CTRL: INPUT INFO: Rs = 01 Rt = 02 Immediate = 0003 ALUCODE = 04 Rd = 01
CTRL: RAM INFO DATA READ: 03002210
CTRL : instruction word: 10220003, opcode: 000100, funccode: 000011
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: INFO: ADDIU called
CTRL: INPUT INFO: Rs = 01 Rt = 02 Immediate = 0003 ALUCODE = 04 Rd = 01
CTRL: RAM INFO DATA READ: 03002210
CTRL : instruction word: 10220003, opcode: 000100, funccode: 000011
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: INFO: ADDIU called
CTRL: INPUT INFO: Rs = 01 Rt = 02 Immediate = 0003 ALUCODE = 04 Rd = 01
CTRL: RAM INFO DATA READ: 03002210
CTRL : instruction word: 10220003, opcode: 000100, funccode: 000011
IIIIIIIIII type
DECODE: Decoding is done, enterring parameter assginment
CTRL: INFO: ADDIU called
CTRL: INPUT INFO: Rs = 01 Rt = 02 Immediate = 0003 ALUCODE = 04 Rd = 01
PC: not reset or halt
FSM: EXEC2
CTRL: Instruction again: 00010000001000100000000000000011, FUNCCODE: 000011
CTRL: Here is the result =         10
CTRL: Instruction again: 00010000001000100000000000000011, FUNCCODE: 000011
CTRL: Here is the result =         10
FSM: FETCH
PC: not reset or halt
PC: current value before increment: bfc00008
CTRL: Here is the result =         10
CTRL: Read instruction at RAM address: bfc00008
CTRL: Here is the result =         10
CTRL: Read instruction at RAM address: bfc0000c
PC: not reset or halt
FSM: EXEC1
CTRL: RAM INFO DATA READ: 21100300
CTRL : instruction word: 00031021, opcode: 000000, funccode: 100001
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: RAM INFO DATA READ: 21100300
CTRL : instruction word: 00031021, opcode: 000000, funccode: 100001
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: RAM INFO DATA READ: 21100300
CTRL : instruction word: 00031021, opcode: 000000, funccode: 100001
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: RAM INFO DATA READ: 21100300
CTRL : instruction word: 00031021, opcode: 000000, funccode: 100001
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
FSM: EXEC2
PC: not reset or halt
CTRL: Instruction again: 00000000000000110001000000100001, FUNCCODE: 100001
CTRL: Here is the result =         10
CTRL: Instruction again: 00000000000000110001000000100001, FUNCCODE: 100001
CTRL: Here is the result =          0
CTRL: Instruction again: 00000000000000110001000000100001, FUNCCODE: 100001
CTRL: Here is the result =          0
CTRL: Instruction again: 00000000000000110001000000100001, FUNCCODE: 100001
CTRL: Here is the result =         20
PC: not reset or halt
PC: current value before increment: bfc0000c
FSM: FETCH
CTRL: Here is the result =         20
CTRL: Read instruction at RAM address: bfc0000c
CTRL: Here is the result =         20
CTRL: Read instruction at RAM address: bfc00010
FSM: EXEC1
PC: not reset or halt
CTRL: RAM INFO DATA READ: 21104400
CTRL : instruction word: 00441021, opcode: 000000, funccode: 100001
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: RAM INFO DATA READ: 21104400
CTRL : instruction word: 00441021, opcode: 000000, funccode: 100001
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
PC: not reset or halt
FSM: EXEC2
CTRL: Instruction again: 00000000010001000001000000100001, FUNCCODE: 100001
CTRL: Here is the result =         20
CTRL: Instruction again: 00000000010001000001000000100001, FUNCCODE: 100001
CTRL: Here is the result =         20
CTRL: Instruction again: 00000000010001000001000000100001, FUNCCODE: 100001
CTRL: Here is the result =         30
FSM: FETCH
PC: not reset or halt
PC: current value before increment: bfc00010
CTRL: Here is the result =         30
CTRL: Read instruction at RAM address: bfc00010
CTRL: Here is the result =         30
CTRL: Read instruction at RAM address: bfc00014
PC: not reset or halt
FSM: EXEC1
CTRL: RAM INFO DATA READ: 21104200
CTRL : instruction word: 00421021, opcode: 000000, funccode: 100001
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: RAM INFO DATA READ: 21104200
CTRL : instruction word: 00421021, opcode: 000000, funccode: 100001
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: RAM INFO DATA READ: 21104200
CTRL : instruction word: 00421021, opcode: 000000, funccode: 100001
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
FSM: EXEC2
PC: not reset or halt
CTRL: Instruction again: 00000000010000100001000000100001, FUNCCODE: 100001
CTRL: Here is the result =         30
CTRL: Instruction again: 00000000010000100001000000100001, FUNCCODE: 100001
CTRL: Here is the result =         40
CTRL: Instruction again: 00000000010000100001000000100001, FUNCCODE: 100001
CTRL: Here is the result =         40
CTRL: Instruction again: 00000000010000100001000000100001, FUNCCODE: 100001
CTRL: Here is the result =         60
PC: not reset or halt
PC: current value before increment: bfc00014
FSM: FETCH
CTRL: Here is the result =         60
CTRL: Read instruction at RAM address: bfc00014
CTRL: Here is the result =         60
CTRL: Read instruction at RAM address: bfc00018
FSM: EXEC1
PC: not reset or halt
CTRL: RAM INFO DATA READ: 08000000
CTRL : instruction word: 00000008, opcode: 000000, funccode: 001000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
Detect JR in EXEC 1, Rs:  0
CTRL: RAM INFO DATA READ: 08000000
CTRL : instruction word: 00000008, opcode: 000000, funccode: 001000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
Detect JR in EXEC 1, Rs:  0
CTRL: RAM INFO DATA READ: 08000000
CTRL : instruction word: 00000008, opcode: 000000, funccode: 001000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
Detect JR in EXEC 1, Rs:  0
CTRL: RAM INFO DATA READ: 08000000
CTRL : instruction word: 00000008, opcode: 000000, funccode: 001000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
Detect JR in EXEC 1, Rs:  0
PC: not reset or halt
FSM: EXEC2
CTRL: Instruction again: 00000000000000000000000000001000, FUNCCODE: 001000
Arrived at jr: 0000003c, Rs:  0
CTRL: Here is the result =          0
CTRL: Instruction again: 00000000000000000000000000001000, FUNCCODE: 001000
Arrived at jr: 00000000, Rs:  0
CTRL: Here is the result =          0
FSM: FETCH
PC: not reset or halt
PC: current value before increment: bfc00018
PC: In jump_r
CTRL: Here is the result =          0
CTRL: Read instruction at RAM address: bfc00018
CTRL: Here is the result =          0
CTRL: Read instruction at RAM address: bfc0001c
PC: not reset or halt
FSM: EXEC1
CTRL: RAM INFO DATA READ: 00000000
CTRL : instruction word: 00000000, opcode: 000000, funccode: 000000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: SLL SHAMT 00, Instruction:00000000, FUNCCODE:000000
HHHHHHHHH ALU CODE 00
CTRL: RAM INFO DATA READ: 00000000
CTRL : instruction word: 00000000, opcode: 000000, funccode: 000000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: SLL SHAMT 00, Instruction:00000000, FUNCCODE:000000
HHHHHHHHH ALU CODE 00
FSM: EXEC2
PC: not reset or halt
CTRL: Instruction again: 00000000000000000000000000000000, FUNCCODE: 000000
CTRL: INFO: Currently in the SLL statement
CTRL: Here is the result =          0
PC: not reset or halt
PC: current value before increment: bfc0001c
PC: In r_delay: 00000000
FSM: FETCH
CTRL: Here is the result =          0
CTRL: Read instruction at RAM address: bfc0001c
CTRL: Here is the result =          0
CTRL: Read instruction at RAM address: 00000000
FSM: EXEC1
CTRL: RAM INFO DATA READ: 00000000
CTRL : instruction word: 00000000, opcode: 000000, funccode: 000000
RRRRRRRRRR type
DECODE: Decoding is done, enterring parameter assginment
CTRL: SLL SHAMT 00, Instruction:00000000, FUNCCODE:000000
HHHHHHHHH ALU CODE 00
FSM: EXEC2
CTRL: Instruction again: 00000000000000000000000000000000, FUNCCODE: 000000
CTRL: INFO: Currently in the SLL statement
CTRL: Here is the result =          0
FSM: FETCH
CTRL: Here is the result =          0
CTRL: Read instruction at RAM address: 00000000
FSM: HALT
FSM: HALT
TB : finished; running=0
TB : INFO : register_v0=0000003c
test/0-MIPS_tb_RAM/mips_cpu_bus_tb.v:97: $finish called at 31500 (10ps)
CPU : OUT   :         60
